lib/puppet/module_tool/applications/unpacker.rb

```diff
@@ -93,7 +93,12 @@ def move_into(dir)
       # @api private
       # @return [String] path to temporary unpacking location
       def tmpdir
-        @dir ||= Dir.mktmpdir('tmp-unpacker', Puppet::Forge::Cache.base_path)
+        base_path = nil
+        if !Puppet::Util::Platform.windows?
+          base_path = Puppet::Forge::Cache.base_path
+        end
+
+        @dir ||= Dir.mktmpdir('tmp-unpacker', base_path)
```

Comment:
I suggest putting a comment in that clarifies the behavior of the "mktmpdir" method when a nil destination path is supplied.